library-model/library-model-generator-integration-test/src/test/java/com/uber/nullaway/libmodel/LibraryModelIntegrationTest.java

```diff
@@ -0,0 +1,76 @@
+package com.uber.nullaway.libmodel;
+
+import com.google.errorprone.CompilationTestHelper;
+import com.uber.nullaway.NullAway;
+import java.util.Arrays;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class LibraryModelIntegrationTest {
+
+  @Rule public final TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+  private CompilationTestHelper compilationHelper;
+
+  @Before
+  public void setup() {
+    compilationHelper = CompilationTestHelper.newInstance(NullAway.class, getClass());
+  }
+
+  @Test
+  public void libraryModelNullableReturnsTest() {
+    compilationHelper
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:JarInferEnabled=true",
+                "-XepOpt:NullAway:JarInferUseReturnAnnotations=true"))
```

Comment:
I agree with @yuxincs that overloading the meaning of the `JarInfer[...]` CLI options for NullAway for this is bad practice, specially because other than the Android SDK, JarInfer no longer uses `.astubx` but actually modifies the bytecode of jars to add annotations (so this options aren't used with JarInfer except for that Android SDK jar).

It can certainly be a follow up PR, but I'd be in favor of rethinking these CLI flags. Maybe `-XepOpt:NullAway:JarInferEnabled` is actually something like `-XepOpt:NullAway:ScanClassPathForAstubXModels`? Is there a better name? A nicer way to specify which `.astubx` files we want to load without breaking things in our build internally? (I think there is! We only really care about that Android SDK astubx file right now, so it's not like this changes for each target like it used to, cc: @yuxincs )

Reply:
I am in favor of changing these, but maybe in a follow-up PR.  I don't think we need to go as far as keeping the old flag but deprecating it, as long as we document properly in release notes.  But we should probably check as to what this breaks (if anything) internally at Uber.

@akulk022 can you open an issue on renaming these flags?

Reply:
I've created the issue https://github.com/uber/NullAway/issues/940